src/cryptoadvance/specter/templates/device/device.jinja

```diff
@@ -95,5 +100,37 @@
 				}
 			}, false);
 		});
+
+		async function togglePassphrase(){
+        	let devices = await enumerate('{{ device.device_type }}');
+			if(devices == null || devices.length == 0){
+				return;
+			}
+			let device = await selectDevice(devices);
```

Comment:
It's very confusing when you disable the passphrase and GUI is asking for a passphrase.
Let's add an optional argument in `selectDevice` function - `passphrase=null`. If we don't care about the password or we know what password we want to use - we can pass it to the function and the GUI window will not appear, and password from the argument will be used.
Then here we can provide an empty passphrase and avoid a passphrase window.